fb-messenger-dark.css

```diff
@@ -51,6 +51,12 @@ input::-webkit-input-placeholder {
 ._33p7 {
 	background-color: rgba(30, 30, 30, 0.9)	!important;
 }
+/* highlighted search results */
+.__in {
+	background-color: rgba(0, 0, 0, 0.3);
+	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
+	padding: 1px 0 2px 0;
```

Comment:
Is this padding change really necessary? Clearly the original is what Facebook intends this to look like… /s

No, but seriously, I don't think this style should be fixing Facebook bugs, no matter how tiny they are.

Reply:
It's necessary for me to maintain my sanity lol

![messenger-bubble-padding-comparison](https://user-images.githubusercontent.com/5691865/37065102-aa4405d0-21f3-11e8-86d0-dd5dc2792a61.gif)

But seriously I don't consider that a bug—just an imperfection—and even if it were an actual bug, I highly doubt they'd ever bother fixing it. I [reported](https://github.com/auscompgeek/fb-messenger-dark/pull/18#issuecomment-298829333) the scrollbar in the input field bug to them 10 months ago, received no response and that bug still exists today.

Reply:
I would consider it to be a bug, as it goes into the space of the above line if the message is wrapped.